lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -48,11 +48,11 @@ echo -e "$check (Potentially stale) lighthouse-dt-bundle copied."
 cp -r dist/dt-report-resources/* "$fe_lh_dir"
 echo -e "$check Report resources copied."
 
-# copy locale JSON files
-lh_locales_dir="lighthouse-core/lib/i18n/locales/"
-fe_locales_dir="$frontend_dir/audits_worker/lighthouse/locales/"
+# copy locale JSON files (but not the .ctc.json ones)
+lh_locales_dir="lighthouse-core/lib/i18n/locales"
+fe_locales_dir="$frontend_dir/audits_worker/lighthouse/locales"
 
-cp -r "$lh_locales_dir" "$fe_locales_dir"
+find $lh_locales_dir -name '*.json' ! -name '*.ctc.json'  -exec cp {} "$fe_locales_dir" \;
```

Comment:
fine as long as the folder is guaranteed to exist. might suggest a quick `mkdir -p $fe_locales_dir` to be explicit.

Reply:
good call, especially while it's not landed.